pgmpy/tests/test_models/test_FunctionalBayesianNetwork.py

```diff
@@ -162,6 +170,146 @@ def test_simulate_different_distributions(self):
         self.assertTrue(np.all(samples["lognormal"] > 0))
         self.assertTrue(np.all(samples["gamma"] > 0))
 
+    def test_svi_fit_normal(self):
```

Comment:
Can we also add a test using the dataset that I had used in the prototype? That is a more realistic scenario with different types of variables connections (v-structures, confounding, etc.). It would help us in making sure that we are not missing something.